src/ring_theory/dedekind_domain/ideal.lean

```diff
@@ -757,6 +757,66 @@ begin
       (lt_top_iff_ne_top.mpr hJ) },
 end
 
+section gcd
```

Comment:
I wonder why this section is still in the diff. Would bad things happen when merging?

Reply:
I think bors is smart enough to handle this: compare [the PR diff](https://github.com/leanprover-community/mathlib/commit/6fa3fc3e974a120f79c4327636da8208d6a01847) and [the bors squashed commit](https://github.com/leanprover-community/mathlib/pull/17289/files). So merging master is not strictly necessary, only makes the PR easier to review.

Reply:
In fact, I think it's entirely handled by git itself: as I understand it, it will construct the diff between the most common ancestor of `master` and the tip of the branch, and any changes that are identical between the branches are automatically included.

I'll push a rebased version to help you :)

Reply:
Yeah thanks, I think git can indeed do that. I think the Github PR "Files Changed" view is not ideal because it simply shows the diff between the most common ancestor of `master` and the tip of the branch, without including those identical changes in the `master` part. It would work if bors doesn't squash merge, because then the common ancestor would already include the identical changes. It would be nice if Github could be configured to show the diff as if `master` has been merged into the branch when there is no conflicts; this should be the same as the diff in the commit that bors would push.